src/SOS/Strike/util.cpp

```diff
@@ -3225,21 +3277,64 @@ size_t FunctionType (size_t EIP)
     return (size_t) pMD;
 }
 
-#ifndef FEATURE_PAL
-
 //
 // Gets version info for the CLR in the debuggee process.
 //
-BOOL GetEEVersion(VS_FIXEDFILEINFO *pFileInfo)
+BOOL GetEEVersion(VS_FIXEDFILEINFO* pFileInfo, char* fileVersionBuffer, int fileVersionBufferSize)
```

Comment:
Nit: We should clarify the arg name or use a comment to say if this is size in bytes or size in chars.

Reply:
Done